src/src/PluginStructs/P128_data_struct.h

```diff
@@ -2255,12 +2266,20 @@ const uint8_t PROGMEM ftv_gamma8[] = {
   #  define FEATURE NeoGrbFeature
 # endif // if defined GRB
 
-# define  NUMPixels (sizeof(ftv_colors) / sizeof(ftv_colors[0]))
+# if P128_ENABLE_FAKETV
+#  define NUMPixels (sizeof(ftv_colors) / sizeof(ftv_colors[0]))
+# else // if P128_ENABLE_FAKETV
+#  define NUMPixels (26000 / 8)
```

Comment:
Why do you need NUMPixels anyway?
Second, `sizeof(ftv_colors[0]))` == 1 , not 8 ;)

Reply:
It's also used by another effect as a random seed... (don't ask me, it was already there 🥲)

Reply:
doesn't look very random to me ;)